### ISSUE TITLE
using beertap costs money

### ISSUE THREAD

**Issue by SwissalpS** [pandorabox-io/in-game#265]
When players currently use a beertap, they get free beer.
What about deducting a small amount from bank account?

I'm wondering if anybody thinks that would be fun.

**Comment by Klaranth** [comment]
Let them pay for the glass.

**Comment by OgelGames** [comment]
Nah, why pay when you don't consume any beer, it's just an illusion that makes you think you got free beer :P

**Comment by S-S-X** [comment]
Could do something similar to soda machine which requires coin, hit with coin to get your beer.

Requiring actual drink barrels and co2 tank would be nice but not very simple change, should not be limited to beer anymore but provide whatever is in barrel.

**Comment by SwissalpS** [comment]
that would mean you buy beer + glass
